iac-extensions/arm/src/test/resources/checks/ResourceSpecificAdminAccountCheck/Microsoft.ContainerRegistry_registries.bicep

```diff
@@ -0,0 +1,22 @@
+resource acrAdminUserEnabled 'Microsoft.ContainerRegistry/registries@2021-09-01' = {
+  properties: {
+    adminUserEnabled: true // Noncompliant{{Make sure that enabling an administrative account or administrative permissions is safe here.}}
+//  ^^^^^^^^^^^^^^^^^^^^^^
+  }
+}
+
+resource acrAdminUserDisabled 'Microsoft.ContainerRegistry/registries@2021-09-01' = {
+  properties: {
+    adminUserEnabled: false
+  }
+}
+
+resource acrAdminInvalidValue 'Microsoft.ContainerRegistry/registries@2021-09-01' = {
+  properties: {
+    adminUserEnabled: 'true'
+  }
+}
+
```

Comment:
Nitpick. You can add the case when `adminUserEnabled ` is not a string literal

Reply:
Same question here.